### ISSUE TITLE
Improve UI of file upload progress

### ISSUE THREAD

**Issue by sffc** [octave-online/octave-online-server#2]
There is currently limited indication on the progress of a file upload or when the file upload is complete, particularly when on a slow connection.  The following error message also gets spammed.

```
If your file is large and causes you to exceed your space limit
(51200 KiB), the file may be incomplete.
```

A better UI would be to disable the command prompt and show a progress bar on the progress of the file upload.

**Comment by Abhirup-99** [comment]
If this issue is open, would like to work on it.

**Comment by sffc** [comment]
Yes, this issue is still open.  #18 is another bug involving file uploads.

I wrote [socketio-file-upload](https://www.npmjs.com/package/socketio-file-upload) for Octave Online Server several years ago.  By today's standards, it is out-dated.  A modern module for file uploads would probably be written in TypeScript and do a better job of tracking file upload progress.  Alternatively, we can keep doing piecemeal improvements to socketio-file-upload.

**Comment by Abhirup-99** [comment]
@sffc I am familiar with both Typescript and socketio. So, can start with a piecemeal approach of the UI and consequently the library or even the library itself. I am more inclined towards fixing the Ui and library for the time being and then migrating the library at a future date? What do you think?

**Comment by sffc** [comment]
Continuing to improve socketio-file-upload to enable this feature and fix #18 sounds great to me. 👍
